From c2718d3567aecb6e266706093b8166e8014d9a6a Mon Sep 17 00:00:00 2001
From: Johannes Thoma <johannes@johannesthoma.com>
Date: Wed, 18 Mar 2020 16:05:37 +0100
Subject: [PATCH] drbd_receiver: initalize iter

Probably a false positive of the MS compiler: it complains
about iter being uninitialized.
---
 drbd/drbd_receiver.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drbd/drbd_receiver.c b/drbd/drbd_receiver.c
index 115eb7c..748364e 100644
--- a/drbd/drbd_receiver.c
+++ b/drbd/drbd_receiver.c
@@ -2051,6 +2051,7 @@ static int recv_dless_read(struct drbd_peer_device *peer_device, struct drbd_req
 	bio = req->master_bio;
 	D_ASSERT(peer_device->device, sector == bio->bi_iter.bi_sector);
 
+	iter = bio->bi_iter;
 	bio_for_each_segment(bvec, bio, iter) {
 		void *mapped = ((u8*)kmap(bvec.bv_page)) + bvec.bv_offset;
 		expect = min_t(int, data_size, bvec.bv_len);
-- 
2.17.0

